docs/getting-started.md

```diff
@@ -2,12 +2,21 @@
 
 *If you've previously installed gulp globally, run `npm rm --global gulp` before following these instructions.*
```

Comment:
I think it should go here.  In the part below, I think you want people focusing on npm and node installation.